sdk/eventhubs/tests.yml

```diff
@@ -0,0 +1,5 @@
+trigger: none
+
+steps:
+  - pwsh: |
+      Write-Host "No-Op"
```

Comment:
Is there any reason to add these placeholders? Why not add them when we want to add the tests.

Reply:
removed